src/prebid.js

```diff
@@ -113,7 +113,12 @@ function resetPresetTargeting() {
   if (isGptPubadsDefined()) {
     window.googletag.pubads().getSlots().forEach(slot => {
       pbTargetingKeys.forEach(function(key){
-        slot.setTargeting(key,null);
+				// Reset Only Registerd dfp-adunits 
+				$$PREBID_GLOBAL$$.adUnits.find(function (unit) {
+					if(unit.code === slot.getSlotElementId()){
```

Comment:
`unit.code` should also check against `slot.getAdUnitPath()`. See https://github.com/prebid/Prebid.js/blob/master/src/prebid.js#L124 for a similar example